src/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var $ = require('jquery')
+
+module.exports = React.createClass({
+    componentDidMount() {
+        var self = this;
```

Comment:
You could get rid of this variable by using arrow function below (which would preserve `this`): `.done((response) => {…})`. But I guess this is a matter of personal taste, no need to change this.

Reply:
Nice, that's pretty convenient. I've changed that.